kernels/wmma_extension.h

```diff
@@ -29,6 +29,18 @@ class fragment<matrix_a, 16, 8, 256, experimental::precision::b1, row_major>
 template <>
 class fragment<matrix_b, 16, 8, 256, experimental::precision::b1, col_major>
     : public __frag_base<experimental::precision::b1, 32, 2> {};
+
+#if defined(__GFX9__) || defined(__GFX12__) || __CUDA_ARCH__ >= 890
```

Comment:
```suggestion
#if __CUDA_ARCH__ >= 890
```
The `__frag_base` class is nvidia-only